index.html

```diff
@@ -1,11 +1,162 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Homework 1</title>
   </head>
   <body>
-    <h1>Start developing here</h1>
+    <header class="header">
+      <a class="header__logo logo" href="./index.html">WebStudio</a>
+
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item is-active">
+            <a class="nav__link" href="#">Студія</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Портфоліо</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Контакти</a>
+          </li>
+        </ul>
+      </nav>
+      <div class="header__contacts contacts">
+        <a class="contacts__email is-active" href="#">info@devstudio.com</a>
+        <p class="contacts__number">+38 096 111 11 11</p>
+      </div>
+    </header>
+    <main class="main">
+      <section class="main__banner banner">
+        <h1 class="banner__title">Ефективні рішення для вашого бізнесу</h1>
+        <button class="banner__button">Замовити послугу</button>
+      </section>
+
+      <section class="main__features features">
+        <ul class="features__list">
+          <li class="features__item feature">
+            <h3 class="feature__title">УВАГА ДО ДЕТАЛЕЙ</h3>
+            <p class="feature__text">
+              Ідейні міркування, і навіть початок повсякденної роботи з
+              формування позиції.
+            </p>
+          </li>
+
+          <li class="features__item feature">
+            <h3 class="feature__title">ПУНКТУАЛЬНІСТЬ</h3>
+            <p class="feature__text">
+              Завдання організації, особливо рамки і місце навчання кадрів тягне
+              у себе.
+            </p>
+          </li>
+
+          <li class="features__item feature">
+            <h3 class="feature__title">ПЛАНУВАННЯ</h3>
+            <p class="feature__text">
+              Так само консультація з широким активом значною мірою зумовлює.
+            </p>
+          </li>
+
+          <li class="features__item feature">
+            <h3 class="feature__title">СУЧАСНІ ТЕХНОЛОГІЇ</h3>
+            <p class="feature__text">
+              Значимість цих проблем настільки очевидна, що реалізація планових
+              завдань.
+            </p>
+          </li>
+        </ul>
+      </section>
+
+      <section class="main__services services">
+        <h2 class="services__title title">Чим ми займаємося</h2>
+        <ul class="services__list">
+          <li class="services__item service">
+            <img
+              class="service__image"
+              src="./images/services--1.jpg"
+              alt="First image in services"
+              width="370px"
```

Comment:
**`width`** attribute is specified in pixels, but we do not write  **`px`** here
```
<img
  class="service__image"
  src="./images/services--1.jpg"
  alt="First image in services"
  width="370" 
/>
...
```

[READ MORE](https://www.w3schools.com/tags/att_img_width.asp)